### PR TITLE
giabalanai info.json fixed.

### PR DESCRIPTION
**Pull request by 3araht**
## Description

info.json fix.
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/giabalanai/info.json [hunk 1]
```diff
@@ -25,12 +150,11 @@
         "midi": true,
         "extrakey": true,
         "encoder": true,
+        "encoder_map": true,
```
Comment (zvecr) [inline keyboards/giabalanai/info.json:153]: `encoder_map` is never allowed to be enabled at the keyboard level.

All `rules.mk` changes regarding `encoder_map` will also need to be reverted.
```suggestion
```

**Review by zvecr — CHANGES_REQUESTED** [review]

File: keyboards/giabalanai/keymaps/default_giabarinaix2/rules.mk [hunk 2]
```diff
@@ -1,3 +1 @@
 ENCODER_ENABLE = no        # encoder on mute button
-
```
Comment (drashna) [inline keyboards/giabalanai/keymaps/default_giabarinaix2/rules.mk:2]: encodermap not being used on thes keymaps?

Comment (3araht) [reply keyboards/giabalanai/keymaps/default_giabarinaix2/rules.mk:2]: No, encodermap is not used in this keymap.
I forgot to disable it in giabarinaix2led keymap. I added it there as well.

**Comment by drashna** [comment]
Changes in data driven (info.json) structure need to go through develop, right now.  Retargeted, but yo ay need to rebase

**Comment by 3araht** [comment]
> Changes in data driven (info.json) structure need to go through develop, right now. Retargeted, but yo ay need to rebase

Uh oh... I don't know how to...
I tried some but they didn't work well.

I rebased it to qmk/develop after I fetched the latest qmk (0.22.10), then it showed errors when I push the branch.
So I undid the rebase, and then I pulled the branch first. 
However, it showed everything is up to date...

**Review by drashna — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by 3araht** [comment]
I have no clue how to fix this.
```
This branch has conflicts that must be resolved
Only those with [write access](https://docs.github.com/articles/what-are-the-different-access-permissions) to this repository can merge pull requests.
Conflicting files
keyboards/giabalanai/info.json
keyboards/giabalanai/rules.mk
```
The changes I made to those files are the actual modification which I'd like to push...

**Pull request closed without merging**